pyproject.toml

```diff
@@ -11,9 +11,9 @@ python_version = 3.12
 exclude = [".venv/*"]
 
 [tool.pytest.ini_options]
-addopts = "-ra --cov"
+addopts = "-ra"
```

Comment:
removing coverage to allow for running `e2e.py` only, if you add --cov it will run all of the tests that start with `test_`

Reply:
Sounds good yeah, think I did the same change for azure-search-openai-demo